libs/features/personalization/personalization.js

```diff
@@ -735,12 +744,18 @@ export function handleFragmentCommand(command, a) {
   return false;
 }
 
+export const handleGlobalNavCommands = (commands, manifestId, rootEl) => {
+  handleCommands(commands, manifestId, rootEl);
+  deleteMarkedEls(rootEl);
+};
```

Comment:
I think instead we should export `handleCommands` and `deleteMarkedEls`.  This will give globalnav code more flexibility to handle specialized cases.

Reply:
I've exported `handleCommands` and `deletedMarkedEls`, and removed `handleGlobalNavCommands`.